### PR TITLE
docs: fix show hide event readability and make controls expanded as d…

### PR DESCRIPTION
**Pull request by shaharzil**
https://monday.monday.com/boards/3532714909/pulses/6494820733

### CODE REVIEW

File: packages/core/src/components/Dialog/__stories__/Dialog.stories.js [hunk 1]
```diff
@@ -19,14 +20,30 @@ const metaSettings = createStoryMetaSettingsDecorator({
   actionPropsArray: [] // List the component's actions here
 });
 
+const showHideArgTypes = {
```
Comment (talkor) [inline packages/core/src/components/Dialog/__stories__/Dialog.stories.js:23]: BTW have you tried using the `enumPropNamesArray`?

Comment (shaharzil) [reply packages/core/src/components/Dialog/__stories__/Dialog.stories.js:23]: yes, it is not doing anything that helps with the readability :(

File: packages/core/.storybook/preview.tsx [hunk 2]
```diff
@@ -45,7 +45,8 @@ const fontLoader = async () => ({
 const preview: Preview = {
   parameters: {
     controls: {
-      sort: "alpha"
+      sort: "alpha",
+      expanded: true
```
Comment (YossiSaadi) [inline packages/core/.storybook/preview.tsx:49]: 🎉

File: packages/core/src/components/Dialog/__stories__/Dialog.mdx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Canvas, Meta } from "@storybook/blocks";
+import { Meta } from "@storybook/blocks";
```
Comment (YossiSaadi) [inline packages/core/src/components/Dialog/__stories__/Dialog.mdx:1]: 💪🏼

File: packages/core/src/components/Dialog/__stories__/Dialog.stories.js [hunk 4]
```diff
@@ -19,14 +20,30 @@ const metaSettings = createStoryMetaSettingsDecorator({
...
+  options: Object.values(HideShowEvent),
+  control: {
+    type: "multi-select"
+  },
+  table: {
+    type: {
+      summary: Object.values(HideShowEvent).join(" | ")
+    }
+  }
+};
```
Comment (YossiSaadi) [inline packages/core/src/components/Dialog/__stories__/Dialog.stories.js:33]: definitely should be (in a future enhancement, maybe create a task?) a common util, or as part of the `createMeta`

**Review by YossiSaadi — APPROVED** [review]

File: packages/core/src/components/Dialog/__stories__/Dialog.stories.js (hunk 4 above)
Comment (shaharzil) [reply packages/core/src/components/Dialog/__stories__/Dialog.stories.js:33]: I'll create a task for it in the backlog

**Pull request merged**